app/api/probe_dock/project_versions_api.rb

```diff
@@ -26,6 +26,8 @@ def current_organization
             Organization.active.where(normalized_name: params[:organizationName].to_s.downcase).first!
           elsif params[:projectId].present?
             Organization.active.joins(:projects).where('projects.api_id = ?', params[:projectId].to_s).first!
+          elsif params[:testId].present?
+            ProjectTest.where(api_id: params[:testId]).first!.project.organization
```

Comment:
This makes 2 queries and loads data that will not be used. The relation should start from Organization and make the necessary joins like the other queries in this method. It also allows you to use `Organization.active` which makes sure that it only works for active organizations. Please also update `ResultsApi#current_organization` to use this pattern.